src/test/java/seedu/address/testutil/TeachingAssistantBuddyBuilder.java

```diff
@@ -4,9 +4,10 @@
 import seedu.address.model.module.Module;
 
 /**
- * A utility class to help with building Addressbook objects.
+ * A utility class to help with building TeachingAssistantBuddy objects.
  * Example usage: <br>
- *     {@code AddressBook ab = new AddressBookBuilder().withPerson("John", "Doe").build();}
+ *     {@code TeachingAssistantBuddy ab = new TeachingAssistantBuilder()
```

Comment:
can change the variable name `ab` to `tab` here?

Reply:
Thanks for pointing out! Will fix